### ISSUE TITLE
Hierarchical probabilistic neural network language model

### ISSUE THREAD

**Issue by tarohi24** [tarohi24/literature#35]
AISTAT. 2005.

**hierarchical softmax**
